protocol/avdl/keybase1/teams.avdl

```diff
@@ -18,6 +18,14 @@ protocol teams {
       KID encKID;
   }
 
+  record TeamMembers {
+    array<string> owners;
+    array<string> admins;
+    array<string> writers;
+    array<string> readers;
```

Comment:
Does this conflict with what miles is working on? There's a ticket for this I believe.  Also, even though most members show in lists of the form `"bob"`, there are people like `"bob%3"` which means bob after a reset at sigchain link 3.  By default, it's assumed `"bob%1"`, and you can leave off the `%1`.  We might want a special type for this.